portals/devportal/source/src/app/components/Apis/Details/Environments.jsx

```diff
@@ -321,7 +330,62 @@ function Environments(props) {
                         </Paper>
                     </>
                 )}
-                {!selectedEndpoint && (
+                {api.advertiseInfo && api.advertiseInfo.advertised
```

Comment:
show both production and sandbox endpoints

Reply:
done with https://github.com/wso2/apim-apps/pull/130/commits/b138ea609173d3c2c23316e65e3a4b109fb237f5